frontend/dockerfile/parser/split_command.go

```diff
@@ -12,7 +12,7 @@ func splitCommand(line string) (string, []string, string, error) {
 	var flags []string
 
 	// Make sure we get the same results irrespective of leading/trailing spaces
-	cmdline := tokenWhitespace.Split(strings.TrimSpace(line), 2)
+	cmdline := reWhitespace.Split(strings.TrimSpace(line), 2)
```

Comment:
why is this in this PR?

Reply:
Ah it's on top of #1441

Reply:
would cause a conflict otherwise. lets merge in order